Make the acceleration preview dismissable

mononaut opened the pull request:
Currently after clicking on the "Accelerate" button, there's no way to get rid of the acceleration preview without navigating away from the transaction page.

This PR adds an 'X' button to dismiss the acceleration preview, and slightly reduces the width of the error/success alert banner to make room for it:

<img width="1136" alt="Screenshot 2023-11-25 at 11 48 00 AM" src="https://github.com/mempool/mempool/assets/83316221/f84e9bff-7517-45d1-a69c-9f9afe0452de">

softsimon commented:
For CPFP, Bowtie, Input/output details on the transaction page we all use the Button as a toggle to reveal a new panel.

So wouldn't it be more consistent (and look nicer) if the Accelerator button also is a toggle.

mononaut commented:
yeah, I suppose I was thinking the functionality is more like a "close" button than a show/hide toggle.

but maybe the same style as the other buttons would look better.

mononaut commented:
Rebased on master and changed to the toggle button style:

<img width="1139" alt="Screenshot 2023-12-01 at 3 01 59 PM" src="https://github.com/mempool/mempool/assets/83316221/fb9796c2-63ad-462b-b6ea-4303ce973371">

softsimon reviewed (approved):
tested ACK @ [1fe983a29]
